### PR TITLE
chore(project): update storybook to v5

### PR DESCRIPTION
**Pull request by gergelyke**

### CODE REVIEW

File: .flowconfig [hunk 1]
```diff
@@ -23,5 +23,6 @@ module.name_mapper.extension='md' -> '<PROJECT_ROOT>/flow/WebpackAsset.js.flow'
 module.name_mapper='^baseui' ->'<PROJECT_ROOT>/src'
 module.name_mapper='^examples' ->'<PROJECT_ROOT>/documentation-site/examples'
 log.file=.flow.log
+sharedmemory.hash_table_pow=21
```
Comment (tajo) [inline .flowconfig]: why is this needed?

**Comment by gergelyke** [comment]
going to reopen against the current master

File: .flowconfig (hunk 1 above)
Comment (gergelyke) [reply .flowconfig]: I am not sure :/ the storybook maintainers suggested this
